lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -170,6 +170,11 @@
                 }
             ];
 
+            // set TimeZoneField to listen to CountryField
+            var timeZoneDropdownField = aboutSectionsData[0].fields[6].view;
+            var countryDropdownField = aboutSectionsData[0].fields[5].view;
```

Comment:
Isn't `aboutSectionsData` a JSON object? You can use `JSON.parse(...)` then access the fields by name.